package.json

```diff
@@ -25,6 +25,7 @@
   },
   "homepage": "https://github.com/lob/hapi-rate-limit",
   "dependencies": {
+    "lodash": "^4.13.1"
```

Comment:
Do you intend to use more lodash functions? If not, you can just import the individual function [`lodash.get`](https://www.npmjs.com/package/lodash.get)